src/main/java/com/passbook/sparkeighteen/peristence/entity/TransactionEntity.java

```diff
@@ -0,0 +1,69 @@
+package com.passbook.sparkeighteen.peristence.entity;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.passbook.sparkeighteen.peristence.POJO.TransactionType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "transaction", schema = "public")
+public class TransactionEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "amount")
+    private Float amount;
+
+    @Column(name = "closing_balance")
+    private Float closingBalance;
+
+    @Column(name = "note")
+    private String note;
+
+    @Column(name = "txt_date_time")
+    @NotNull
+    @JsonFormat(pattern = "MMM dd, yyyy hh:mm:ss a", timezone = "Asia/Kolkata")
+    private LocalDateTime time;
+
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @Column(name = "ref_number")
+    private UUID referenceNumber;
```

Comment:
What is this for can you explain the significance?

Reply:
It is reference number to identify particular transaction

Reply:
Why still  no replay no action?

Reply:
removed